backend/ctgov/api/views.py

```diff
@@ -282,65 +320,87 @@ def post(self, request, *args, **kwargs):
         if not last:
             last = 100
 
-        logger.info("Construct filters")
-        filters = construct_filters(request)
-        q_title_acronym = filter_title_acronym(request)
-        q_outcome_measure = filter_outcome_measure(request)
-        q_eligibility_age = filter_eligibility_age(request)
-        q_age_between = filter_age_between(request)
-        q_study_roa = filter_study_roa(request)
-        q_study_status = filter_study_status(request)
-        q_phase = filter_phase(request)
-
-        search_results_all = (
-            SearchStudies.objects.filter(**filters)
-            .filter(q_title_acronym)
-            .filter(q_outcome_measure)
-            .filter(q_eligibility_age)
-            .filter(q_age_between)
-            .filter(q_study_roa)
-            .filter(q_study_status)
-            .filter(q_phase)
-            .all()
-        )
-
-        search_results = search_results_all.values(
-            "status",
-            "brief_title",
-            "nct_id",
-            "condition_name",
-            "intervention_name",
-            "location_name",
-            "study_phase",
-            "sponsor_name",
-            "location_name",
-            "study_brief_desc",
-            "primary_outcome_measures",
-            "secondary_outcome_measures",
-            "study_start_date",
-            "primary_completion_date",
-        )[first:last]
+        # If elastic search configuration is enabled, perform elastic search; else do django search
+        if ENABLE_ELASTIC == "ON":
+            logger.info("Construct Elastic filters")
+            filters = construct_elastic_filters(request, first, last)
+            es = Elasticsearch(["elasticsearch:9200"])
```

Comment:
"elasticsearch" is used to reference the elasticsearch container within Docker. This won't work in the cloud so we will have to make this an environment variable